### PR TITLE
Uploading HK Records

### PR DESCRIPTION
**Pull request by mominr**

### CODE REVIEW

File: WalkingChallenge/AppController.swift [hunk 1]
```diff
@@ -32,6 +32,7 @@ import FacebookCore
 
 class AppController {
   static let shared = AppController()
+  let queue = OperationQueue()
```
Comment (compnerd) [inline WalkingChallenge/AppController.swift:35]: Does this need external visibility?  Why not make it internal?

Comment (mominr) [inline WalkingChallenge/AppController.swift]: For now let's make it internal.

Comment (mominr) [reply WalkingChallenge/AppController.swift:35]: it is by default internal, when no access control is defined.

File: WalkingChallenge/AppController.swift [hunk 2]
```diff
@@ -134,4 +136,38 @@ extension AppController {
...
+              provider.retrieveStepCountForDateRange(interval) { distance in
+                guard distance > 0 else { return }
+                let record = Record(date: Date(), distance: distance, fbid: Facebook.id, source: source)
+                AKFCausesService.createRecord(record: record)
+              }
+              break
+            }
+          }
+          break
+        }
```
Comment (compnerd) [inline WalkingChallenge/AppController.swift:169]: This chaining is pretty annoying.  Can we split this up some how?

**Comment by compnerd** [comment]
Don't think that we need to upload the HK records as we are going to be using HK only

**Pull request closed without merging**